wazo_webhookd/plugins/config/service.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020-2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import threading
+
+
+class ConfigService:
+
+    # share the lock between service instances
+    _lock = threading.Lock()
```

Comment:
What is the usecase to have more than one `ConfigService` instance? And how can we have this setup with our code?
I understand the lock between two requests and I don't care to have it in the instance level vs the init. But I don't understand the comment

Reply:
I updated the comment to make the purpose clearer.

Reply:
Oh my bad, I was focused on the config update instead of root_logger
Thanks